ansible/roles/run-test-target/tasks/test-collection-method.yml

```diff
@@ -127,6 +128,28 @@
         dest: "{{ logs_root }}/integration-test.log"
       delegate_to: localhost
 
+    - name: Move core dump files to core dump directory
+      become: yes
+      shell: |
+        mkdir /tmp/core_dumps
+        mv /tmp/core.out* /tmp/core_dumps || true
+
+    - name: Get core dump file paths
+      become: yes
+      find:
+        paths: "/tmp/core_dumps/"
+      register: core_files
+
+    - name: Fetch each core dump file
+      become: yes
+      fetch:
+        src: "{{ core_file }}"
+        dest: "{{ logs_root }}/"
```

Comment:
Minor suggestion, but it might be worth putting the core dumps into their own subdirectory within the logs root